### PR TITLE
upgrade n8n from 0.67.3 to 0.116.1

### PR DESCRIPTION
**Pull request by LiquidITGuy**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/n8n-io.yml [hunk 1]
```diff
@@ -81,9 +84,7 @@ caproverOneClickApp:
         end: >-
             n8n is deployed and available as $$cap_appname.
 
-            IMPORTANT!! You need to follow these post-installation steps
-
-            First, please change your nginx config according to this issue https://github.com/caprover/one-click-apps/issues/265
```
Comment (LiquidITGuy) [inline public/v4/apps/n8n-io.yml:86]: not needed anymore

**Comment by githubsaturn** [comment]
Thanks!

File: public/v4/apps/n8n-io.yml [hunk 2]
```diff
@@ -19,6 +19,8 @@ services:
         image: n8nio/n8n:$$cap_n8n_version
         volumes:
             - $$cap_appname-data:/root/.n8n
+            - $$cap_appname-data-write:/files
+            - $$cap_appname-docker-daemon:/var/run/docker.sock
```
Comment (githubsaturn) [inline public/v4/apps/n8n-io.yml:23]: Yikes. Just saw this, it should be:
```
            - /var/run/docker.sock:/var/run/docker.sock
```

Comment (githubsaturn) [reply public/v4/apps/n8n-io.yml:23]: I'll make the change

Comment (LiquidITGuy) [reply public/v4/apps/n8n-io.yml:23]: thanks :)

**Pull request merged**